src/spatialdata_plot/pl/basic.py

```diff
@@ -950,8 +950,10 @@ def show(
 
                     if wanted_labels_on_this_cs:
                         if (table := params_copy.table_name) is not None:
-                            colors = sc.get.obs_df(sdata[table], params_copy.color)
-                            if isinstance(colors[params_copy.color].dtype, pd.CategoricalDtype):
+                            keys = [params_copy.color] if not isinstance(params_copy.color, list) else params_copy.color
```

Comment:
I got lost on the type of `params_copy.color`. Is this always a `str`? Or is it sometimes an `Iterable`? Can it be of `len() > 1` here? The tests pass but if we know it's always a string we can write better code.

Reply:
I'm pretty sure it can never be an Iterable, but would like to check more carefully before really relying on this

Reply:
it can never be an interable, it is always either color like or it corresponds to a particular column that then contains the color.